test/api_test.js

```diff
@@ -79,6 +79,16 @@ describe("KintoClient", () => {
         .eql(requestMode);
     });
 
+    it("should keep the default timeout in the child HTTP instance", () => {
+      expect(new KintoClient(sampleRemote).http.timeout).eql(5000);
```

Comment:
This PR is about the documentation so we can discuss that in another one if we think 5 is not enough.